examples/umd-client/index.html

```diff
@@ -32,9 +45,9 @@
        * Provide your MapBox Token
        **/
       const MAPBOX_TOKEN = 'PROVIDE_MAPBOX_TOKEN';
```

Comment:
Do we use maplibre now?

Reply:
The mapbox token was used for different things

- **allow mapbox-gl library to do a renderer**. `maplibre-gl` doesn't need a token for this local operation, and can readily be used offline if need be with a set of downloaded tiles.
- **access mapbox styles from style api (and map tiles)**. MapLibre doesn't have any services. We use the free Carto styles by default which doesn't need a token. If a paid style is to be used, like Stadia maps or MapTiler, an apikey will be part of the url when it's added, so here also we don't need a similar api key.